esphome/components/remote_transmitter/remote_transmitter_esp32.cpp

```diff
@@ -9,13 +9,22 @@ namespace remote_transmitter {
 
 static const char *const TAG = "remote_transmitter";
 
-void RemoteTransmitterComponent::setup() { this->configure_rmt_(); }
+void RemoteTransmitterComponent::setup() {
+  ESP_LOGCONFIG(TAG, "Setting up Remote Transmitter...");
+  this->configure_rmt_();
+}
 
 void RemoteTransmitterComponent::dump_config() {
   ESP_LOGCONFIG(TAG, "Remote Transmitter...");
```

Comment:
Wonder how this got here, I've never noticed it before. haha
```suggestion
  ESP_LOGCONFIG(TAG, "Remote Transmitter:");
```